test/unit/adjustTier_Unit.t.sol

```diff
@@ -1002,6 +1018,64 @@ contract Test_adjustTier_Unit is UnitTestSetup {
         hook.adjustTiers(tierConfigsToAdd, new uint256[](0));
     }
 
+    function  test_adjustTiers_revertIfCannotRemoveTier() public {
+        uint256 initialNumberOfTiers = 1;
```

Comment:
nit> start with 2  (or more) so we're sure there is no 0 tier left scenario? nitnitnit tho

Reply:
nice, done!